Example/Pods/Segmentify/Pod/Classes/ProductSearchModel.swift

```diff
@@ -0,0 +1,148 @@
+//
+//  ProductSearchModel.swift
+//  Pods
+//
+//  Created by Ebru Goksal on 8.01.2020.
+//
+
+import Foundation
+
+public class ProductSearchModel : NSObject, NSCopying, Codable {
+    
+    public var productId:String?
+    public var name:String?
+    public var inStock:Bool?
+    public var url:String?
+    public var mUrl:String?
+    public var image:String?
+    public var imageXS:String?
+    public var imageS:String?
+    public var imageM:String?
+    public var imageL:String?
+    public var imageXL: String?
+    public var category:String?
+    public var categories:[String]?
+    public var brand:String?
+    public var price:Double?
+    public var oldPrice:Double?
+    public var gender:String?
+    public var colors:[String]?
+    public var sizes:[String]?
+    public var labels:[String]?
+    public var noUpdate:Bool?
+    public var params:[String:String]?
+    public var priceText:String?
+    public var oldPriceText:String?
+    public var specialPriceText:String?
+    public var language:String?
+    public var currency:String?
+    public var quantity:String?
+    
+    enum CodingKeys: String, CodingKey {
+      case productId
+      case name
+      case inStock
+      case url
+      case mUrl
+      case image
+      case imageXS
+      case imageS
+      case imageM
+      case imageL
+      case imageXL
+      case category
+      case categories
+      case brand
+      case price
+      case oldPrice
```

Comment:
@ebruersoy where is **specialPrice**?

Reply:
@candanceylan search does not use special price, so I removed specialPriceText, also I removed codable interface , I used it while I was trying to make json maping things but it didn't work. So I removed codable interface and its constructor with decoder.

Reply:
Fair enough :)